server/utils/verifiers/EdanVerifier.ts

```diff
@@ -0,0 +1,426 @@
+import * as COL from '../../collections/interface';
+import * as LOG from '../logger';
+import * as DBAPI from '../../db';
+import * as V from './VerifierBase';
+
+export type EdanVerifierConfig = {
+    collection: COL.ICollection,

```

Comment:
I prefer use of ; instead of  , ... that's the style used elsewhere in the system.  Match and keep consistent?